src/styles/_mixins.scss

```
@@ -0,0 +1,4 @@
+@mixin square {
+  width: $sheet-size;
+  height: $sheet-size;
+}
```

Міксін який не приймає нічого і використовує для розмірів зміння зовні не можна перевикористати і це швидше не міксі квадрат а міксін calendar_cell

```suggestion
@mixin square ${size} {
  width: $size;
  height: $size;
}
```